app/views/admin/snap_applications/_funnel_stage.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="main-content__funnel__stage">
+  <h4><%= current_stage.count %> <%= name %>
+      <%- if previous_stage.present? %>
+        (<%= number_to_percentage((current_stage.count.to_f / previous_stage.count.to_f) * 100, precision: 1) %>)
```

Comment:
possible to throw this line in a method of some sort? feels like it could use a test